src/pal/pal_posix.h

```diff
@@ -126,8 +120,11 @@ namespace snmalloc
      * POSIX systems are assumed to support lazy commit. The build system checks
      * getentropy is available, only then this PAL supports Entropy.
      */
-    static constexpr uint64_t pal_features =
-      LazyCommit | SNMALLOC_PLATFORM_HAS_GETENTROPY;
+    static constexpr uint64_t pal_features = LazyCommit
+#if defined(SNMALLOC_PLATFORM_HAS_GETENTROPY)
+      | SNMALLOC_PLATFORM_HAS_GETENTROPY
```

Comment:
```suggestion
      | Entropy
```

Reply:
ok makes sense was not sure if you wanted to keep `Entropy`